scripts/ai/controllers/CutterController.lua

```diff
@@ -9,9 +10,22 @@ end
 
 function CutterController:getDriveData()
 	self:disableCutterTimer()
+	--- Turns off the cutter, while the driver is waiting for unloading.
+	if self.driveStrategy.isWaitingForUnload and self.driveStrategy:isWaitingForUnload() then 
+		if self.implement:getIsTurnedOn() then 
+			self.vehicle:aiBlock()
+		end
+	end
 	return nil, nil, nil, nil
 end
 
+function CutterController:onLowering()
```

Comment:
Are we sure that every transition out of one of the waiting for unload states will trigger this?

Reply:
Not sure if these cover all possibilities. Was hoping you could maybe check this, as the combine state machine is pretty complex and I didn't want to change anything there.

Reply:
That's why my first implementation for the sprayer was turning on when it started moving, so here, onRaise would set a flag (wasRaised or something) and then in getDriveData, when the state is not waiting for unload, it could call unblock when not turned on.

Reply:
even better...